src/ProductDefects.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\UsedGoods.
+ */
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Used or defective goods user consent checkbox functionality.
+ */
+class ProductDefects {
+
+  /**
+   * Checkbox consent initialization method.
+   */
+  public static function init() {
+    // Logs consent to defective or used product when product is added to cart.
+    add_action('woocommerce_add_to_cart', function () {
+      ProductDefects::logConsent();
+    });
+
+    add_action('wp', function () {
+      $show_value = get_post_meta(get_the_ID(), '_' . Plugin::PREFIX . '_show_product_defects_consent');
+      if ($show_value && $show_value[0] === 'yes') {
+        $show_override = TRUE;
+      }
+      else {
+        $show_override = FALSE;
+      }
```

Comment:
In my opinion this can be written shorter, like:
```suggestion
     $show_override = $show_value && $show_value[0] === 'yes' ? TRUE : FALSE;
```

Reply:
I like the ternaries but I thought it might be clearer this way. I'll change it because well, I don't have strong opinions.

Reply:
How the hell will not work 😄 ... I just came by to add my own final suggestion which is just short this as 
```
$show_override = $show_value && $show_value[0] === 'yes';
```
and that should do it 😉